app/models/setting/provisioning.rb

```diff
@@ -110,7 +121,7 @@ def self.map_pxe_kind
     end
   end
 
-  def self.default_excluded_facts(ignored_interfaces = IGNORED_INTERFACES)
-    ignored_interfaces
+  def self.default_excluded_facts
```

Comment:
do we really want to drop the possibility to override the default?

Reply:
I searched the codebase and I could not find use of this, prove me wrong.

Reply:
I'm fine with dropping it, as far as i can tell this is only used for the default value for `excluded_facts` setting.

Reply:
Yeah that's what I thought. We added this but never used, to me it's rather confusing so I removed it.